content/references/rippled-api/transaction-formats/transaction-types/accountdelete.md

```diff
@@ -0,0 +1,51 @@
+# AccountDelete
+
+[[Source]](https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/DeleteAccount.cpp "Source")
+
+_Requires the [DeletableAccounts amendment](known-amendments.html#deletableaccounts) :not_enabled:_
+
+An AccountDelete transaction deletes an [account from the XRP Ledger](accountroot.html) if possible, sending its XRP to a specified destination account. See [Deletion of Accounts](accounts.html#deletion-of-accounts) for the requirements to delete an account.
+
+## Example {{currentpage.name}} JSON
+
+```json
+{
+    "TransactionType": "AccountDelete",
+    "Account": "rWYkbWkCeg8dP6rXALnjgZSjjLyih5NXm",
+    "Destination": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    "DestinationTag": 13,
+    "Fee": "5000000",
+    "Sequence": 2470665,
+    "Flags": 2147483648
+}
+```
+
+{% include '_snippets/tx-fields-intro.md' %}
+<!--{# fix md highlighting_ #}-->
+
+| Field            | JSON Type        | [Internal Type][] | Description        |
+|:-----------------|:-----------------|:------------------|:-------------------|
+| `Destination`    |  String - [Address][] | Account      | The address of an account to receive any leftover XRP after deleting the sending account. Must be a funded account in the ledger, and must not be the sending account. |
+| `DestinationTag` | Number           | UInt32            | _(Optional)_ Arbitrary [destination tag](source-and-destination-tags.html) that identifies the a hosted recipient or other information to the recipient of the deleted account's leftover XRP. |
```

Comment:
Pick an article. You have "a" and "the" in the definition. Also, the 2nd half of the sentence reads a bit strange:

that identifies a hosted recipient or other information **for** the recipient of the deleted account's leftover XRP.
Or maybe this works as well:
that identifies a hosted recipient or other information **belonging** to the recipient of the deleted account's leftover XRP.

that identifies a hosted recipient or other information belonging to the recipient of the deleted account's leftover XRP.